docs/source/getting_started/install.rst

```diff
@@ -1,76 +1,98 @@
 Installation
 ============
 
-DeepReg is written in Python 3.7. Dependent external libraries provide core IO functionalities and other standard
-processing tools. The dependencies for DeepReg are managed by
-``pip``.
+DeepReg uses in Python 3.7 and external python dependencies are defined in `requirements <https://github.com/DeepRegNet/DeepReg/blob/main/requirements.txt>`__.
+DeepReg primarily supports and is regularly tested with Ubuntu and Debian Linux distributions.
 
-Create a virtual environment
-----------------------------
+There are multiple different methods to install DeepReg:
 
-The recommended method is to install DeepReg in a dedicated virtual
-environment to avoid issues with other dependencies. The conda
-environment is recommended:
+1. Clone `DeepReg`_ and create a virtual environment using `Anaconda`_ / `Miniconda`_ (**recommended**).
+2. Clone `DeepReg`_ and build a docker image using the provided docker file.
+3. Install directly from PyPI release without cloning `DeepReg`_.
 
-`Anaconda <https://docs.anaconda.com/anaconda/install/>`__ /
-`Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__
+Install via Conda
+-----------------
 
-DeepReg primarily supports and is regularly tested with Ubuntu and Debian Linux distributions.
+The recommended method is to install DeepReg in a dedicated virtual
+environment using `Anaconda`_ / `Miniconda`_ to avoid issues with other dependencies.
+
+Please clone `DeepReg`_ first and install DeepReg following the instructions below.
```

Comment:
- Does the `conda create` actually install deepreg? if not, we need to add this;
- Need to add `git clone`
- Need to add `cd DeepReg`
- Is there a way to customise to another conda env name? I can think of multiple workflows that can utilise this, ideally, something like `conda env create -n deepreg0 -f environment_cpu.yml`. If not, add a comment explaining one needs to modify the yml for changing the default name 'deepreg'

Also consider:
"Clone DeepReg repository, create the pre-defined conda environment and install DeepReg following the instructions below. See official documentation for details regarding the conda environment.

Reply:
for the renaming, that's advanced? check conda docs for the renaming.
why people have multiple deepreg env :(